MediaManager/MediaManagerBase.cs

```diff
@@ -341,6 +344,17 @@ protected TimeSpan PreviousPosition
             }
         }
 
+        public VideoSize _videoSize;
+        public VideoSize VideoSize 
```

Comment:
I think move back to IMediaPlayer, since that one will have base class now. Also split up into int and float.